libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -628,6 +644,15 @@ AP_GPS_UBLOX::_parse_gps(void)
                     break;
                 }
             }
+        } else if(_msg_id == MSG_ACK_NACK) {
+            switch(_buffer.ack.clsID) {
+            case CLASS_CFG:
+                switch(_buffer.ack.msgID) {
+                case MSG_CFG_SBAS:
+                    _unconfigured_messages &= ~CONFIG_SBAS;
```

Comment:
If the GPS NACK's a packet and we can't force it thats absolutely fine. But we *should* be failing prearm checks then. If you are trying to use a GPS that can't support using SBAS you can still disable it can't you? (Or really just check that it isn't enabled) It requires the user to set a parameter, but thats absolutely fine for what it buys us.